src/translation/english.c

```diff
@@ -104,7 +104,9 @@ static translation_string all_strings[] = {
     {TR_HOTKEY_SET_BOOKMARK_3, "Set bookmark 3"},
     {TR_HOTKEY_SET_BOOKMARK_4, "Set bookmark 4"},
     {TR_HOTKEY_EDITOR_TOGGLE_BATTLE_INFO, "Toggle battle info"},
-    {TR_HOTKEY_EDIT_TITLE, "Press new hotkey"}
+    {TR_HOTKEY_EDIT_TITLE, "Press new hotkey"},
+    {TR_HOTKEY_DUPLICATE_TITLE, "Hotkey is duplicated"},
+    {TR_HOTKEY_DUPLICATE, "This key combination assigned to action:"}
```

Comment:
Suggested text:
`This key combination is already assigned to the following action:`